src/emulator/controller.ts

```diff
@@ -43,6 +43,7 @@ import { getDefaultDatabaseInstance } from "../getDefaultDatabaseInstance";
 import { getProjectDefaultAccount } from "../auth";
 import { Options } from "../options";
 import { ParsedTriggerDefinition } from "./functionsEmulatorShared";
+import { ExtensionsEmulator } from "./extensionsEmulator";
 
```

Comment:
Can you add a todo at line 335 (github don't let me comment on that line..), we need to show up on that list 
```
lihes-macbookpro2:ext-emu lihes$ firebase emulators:start
i  emulators: Starting emulators: functions, firestore
⚠  functions: The following emulators are not running, calls to these services from the Functions emulator will affect production: auth, database, hosting, pubsub, storage
i  extensions: firebase/delete-user-data@0.1.11 already downloaded...
...
```

Reply:
And also supporting the --only flag? Related note, if someone does `--only extensions`, we need to bring up the functions emulator but probably shouldn't bring up user's functions

Reply:
Good catch - adding a TODO here and filed b/217189992 to track it